### PR TITLE
Add Timeline to Model

### PR DESCRIPTION
**Pull request by tarrekshaban**
This PR does three things:

- Adds a new `<Timeline />` component for use across pages, this component is not tied to the schema of versions
- Adds a new Memoized function in `/model/overview` that loads and processes the timeline history of the model for the `<Timeline />` component
- Upgrades Blueprint from 4.2 to 5.0, which involved updates a few types across files

### CODE REVIEW

File: frontend/src/components/core/Timeline.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Icon } from "@blueprintjs/core"
+import { BlueprintIcons_16Id } from "@blueprintjs/icons/src/generated/16px/blueprint-icons-16.ts"
+import Pill from "./Pill"
+
+interface metadata {
```
Comment (a10y) [inline frontend/src/components/core/Timeline.tsx:5]: Should metadata items also have a name/key?

File: frontend/src/components/core/Timeline.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    icon?: BlueprintIcons_16Id,
+    value: string,
+}
+
+export interface timelineEvent {
```
Comment (a10y) [inline frontend/src/components/core/Timeline.tsx:10]: Not: exported types generally should be capitalized

**Review by a10y — APPROVED** [review]
Looks good! Didn't run because reviewing from phone but the code seems good

File: frontend/src/components/core/Timeline.tsx (hunk 2 above)
Comment (tarrekshaban) [reply frontend/src/components/core/Timeline.tsx:10]: addressed

File: frontend/src/components/core/Timeline.tsx (hunk 1 above)
Comment (tarrekshaban) [reply frontend/src/components/core/Timeline.tsx:5]: added

**Pull request merged**